src/rust/lib/system/web/src/lib.rs

```diff
@@ -503,7 +503,35 @@ pub async fn sleep(duration:Duration) {
 #[cfg(not(target_arch = "wasm32"))]
 pub use async_std::task::sleep;
 
+/// Stores arguments extracted from Location search. e.g. http://localhost/?arg0=value0&arg1=value1.
```

Comment:
Location is written using upper name but it is neither in backtics nor it is a popular name of something. What is this location referring to ?